package.json

```diff
@@ -76,7 +76,7 @@
     "vue": "^2.5.16",
     "vue-template-compiler": "^2.5.16",
     "vue-router": "^3.0.1",
-    "@ionic/core": "^4.6.0"
+    "@ionic/core": "latest"
```

Comment:
this way you expect users to always adopt the latest `@ionic/core` which is likely to be far from reality. You want to set the minimal version that works with this library

Reply:
That's true, I guess even though this is a bleeding edge library, it still makes sense. Thanks